holidayextras/brands
# Fixed safari and IE bugs with product list thumb column height

#### What does this PR do? (please provide any background)
Tweaks some classes and a few markup changes to fix bugs in Safari and IE with unexpected product-list grid wrapping, and incorrect full-height thumbnail sizing.

#### How can this be tested?
* Checkout the corresponding branch in Pattern Library - `globals-product-list-fixes` (see [PR](https://github.com/holidayextras/pattern-library/pull/281))
* `npm link` the brands.
* Fire up IE or Safari and ensure the full-height thumbnails look like the screenshot below and the correct number of product tiles are displayed on each row of the grid example.

#### Any tech debt?
Still some issues in IE 10 and below with flexbox, but this can be fixed with the correct browser prefixes - probably better to automate with [autoprefixer](https://autoprefixer.github.io/) or something similar.

#### Screenshots / Screencast
![image](https://user-images.githubusercontent.com/2961244/27795063-86a0c294-5ffc-11e7-9ec6-6d10c83ff146.png)

#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/11ZSwQNWba4YF2/giphy.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
